Configure

```diff
@@ -756,11 +756,22 @@ while (@argvcopy)
 			$disabled{"shared"} = "forced";
 			$disabled{"threads"} = "forced";
 			}
-		elsif (/^-D(.*)$/)
+		elsif (/^[-\/]D(.*)$/)
 			{
 			push @user_defines, $1;
 			}
-		else	# common if (/^[-+]/), just pass down...
+		elsif (/^\/link$/)
+			{
+			# This is Visual Studio-specific thing.
+			# Consume all options that start with slash, e.g.
+			# /link /subsystem:console,5.1. Yes, it has to
+			# be last or terminated with non-slash option.
+			while (@argvcopy[0] =~ /^\//)
+				{
+				$libs.=shift(@argvcopy) . " ";
+				}
+			}
```

Comment:
Well, it's not worse than having pass-through options and having them mixed with non-pass-through ones. It's kind of like irregular verbs, it doesn't matter if you find them illogical, you have no option not to use them :-) Well, it's somewhat flawed comparison, but consider all the times we said "pass it as additional option at config-time". I'd argue that if we go for `--cflags`, then we probably have to maintain legacy way for substantial time anyway, At least it would be inappropriate to remove it in 1.1.x...